include/af/arith.h

```diff
@@ -322,79 +327,87 @@ namespace af
     /// @{
     /// C++ Interface to create a complex array from a single real array.
     ///
-    /// \param[in] in a real array
-    /// \return the returned complex array
+    /// \param[in] in input array
```

Comment:
I think this is less descriptive than the previous version. The user doesn't know it should be a real array from this parameter description.